tests/migration_test.go

```diff
@@ -96,6 +96,50 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 	var virtClient kubecli.KubevirtClient
 	var err error
 
+	cDandCloudInitServiceAccountConfigMapSecretDownwardAPIKernelBoot := func() *v1.VirtualMachineInstance {
```

Comment:
`cDand`? We need a generic name for all those, I usually call them "config drives" or "config disks", since they all live in `pkg/config` and show up as drives/disks.
Maybe this could be split into 2 functions, like `newFedoraVMIWithKernelBoot()` and `addAllConfigDrivesToVMI(vmi)`?
Also, is there a reason for creating a local function and calling it on the VMI instead of just modifying the VMI directly?